internal/driver/driver.go

```diff
@@ -0,0 +1,147 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2018 Canonical Ltd
+// Copyright (C) 2018-2019 IOTech Ltd
```

Comment:
The general rule is that you should add a copyright file if you're contributing more than a few lines of code (i.e. more than just bug fixes).